provide links to ECMA and ISO standards (#14880)
dotnet/docs: docs/standard/components.md

- [NuGet](/nuget/), Microsoft's package manager for .NET
- Open-source build orchestration tools, such as [CAKE](https://cakebuild.net/) and [FAKE](https://fake.build/)

## Applicable standards

> The C# Language and the Common Language Infrastructure (CLI) specifications are standardized through [Ecma International®](http://www.ecma-international.org/). The first editions of these standards were published by Ecma in December 2001.

Reviewer: ```suggestion
The C# Language and the Common Language Infrastructure (CLI) specifications are standardized through [Ecma International®](https://www.ecma-international.org/). The first editions of these standards were published by Ecma in December 2001.
```

> Subsequent revisions to the standards have been developed by the TC49-TG2 (C#) and TC49-TG3 (CLI) task groups within the Programming Languages Technical Committee ([TC49](http://www.ecma-international.org/memento/tc49.htm)), and adopted by the Ecma General Assembly and subsequently by ISO/IEC JTC 1 via the ISO Fast-Track process.

Reviewer: ```suggestion
Subsequent revisions to the standards have been developed by the TC49-TG2 (C#) and TC49-TG3 (CLI) task groups within the Programming Languages Technical Committee ([TC49](https://www.ecma-international.org/memento/tc49.htm)), and adopted by the Ecma General Assembly and subsequently by ISO/IEC JTC 1 via the ISO Fast-Track process.
```

> ### Latest Standards

Reviewer 2: ```suggestion
### Latest standards
```

> The following official Ecma documents are available for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [CLI](http://www.ecma-international.org/publications/standards/Ecma-335.htm) ([TR-84](http://www.ecma-international.org/publications/techreports/E-TR-084.htm)). These links are direct from Ecma International®:

Reviewer 2: Do you really need the last sentence?

```suggestion
The following official Ecma documents are available for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [CLI](http://www.ecma-international.org/publications/standards/Ecma-335.htm) ([TR-84](http://www.ecma-international.org/publications/techreports/E-TR-084.htm)):
```

> The following official Ecma documents are available for [C#](http://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [CLI](http://www.ecma-international.org/publications/standards/Ecma-335.htm) ([TR-84](http://www.ecma-international.org/publications/techreports/E-TR-084.htm)). These links are direct from Ecma International®:

Reviewer: ```suggestion
The following official Ecma documents are available for [C#](https://www.ecma-international.org/publications/standards/Ecma-334.htm) and the [CLI](https://www.ecma-international.org/publications/standards/Ecma-335.htm) ([TR-84](https://www.ecma-international.org/publications/techreports/E-TR-084.htm)). These links are direct from Ecma International®:
```

> - **The C# Language Standard (version 5.0)**: [ECMA-334.pdf](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-334.pdf)

Reviewer: ```suggestion
- **The C# Language Standard (version 5.0)**: [ECMA-334.pdf](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-334.pdf)
```

> - **The Common Language Infrastructure**: This is avaialble in [pdf](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf) form and [zip](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.zip) form.

Reviewer: ```suggestion
- **The Common Language Infrastructure**: This is avaialble in [pdf](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf) form and [zip](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.zip) form.
```

Reviewer 2: I've added a typo fix in your suggestion as well @Youssef1313.

> - **Information Derived from the Partition IV XML File**: This is available in [pdf](http://www.ecma-international.org/publications/files/ECMA-TR/ECMA%20TR-084.pdf) and [zip](http://www.ecma-international.org/publications/files/ECMA-TR/TR-084.zip) format.

Reviewer: ```suggestion
- **Information Derived from the Partition IV XML File**: This is available in [pdf](https://www.ecma-international.org/publications/files/ECMA-TR/ECMA%20TR-084.pdf) and [zip](https://www.ecma-international.org/publications/files/ECMA-TR/TR-084.zip) formats.
```

> The official ISO/IEC documents are available from the ISO/IEC [Freely Available Standards](https://standards.iso.org/ittf/PubliclyAvailableStandards/) page. These links are direct from that page:

Reviewer 2: The target says Publicly not Freely

```suggestion
The official ISO/IEC documents are available from the ISO/IEC [Publicly Available Standards](https://standards.iso.org/ittf/PubliclyAvailableStandards/) page. These links are direct from that page:
```
